src/Fabulous.XamarinForms/Views/Layouts/FlexLayout.fs

```diff
@@ -47,8 +49,17 @@ module FlexLayout =
 [<AutoOpen>]
 module FlexLayoutBuilders =
     type Fabulous.XamarinForms.View with
-        static member inline FlexLayout<'msg>() =
-            CollectionBuilder<'msg, IFlexLayout, IView>(FlexLayout.WidgetKey, LayoutOfView.Children)
+        static member inline FlexLayout<'msg>(?wrap: FlexWrap) =
+            let wrap =
+                match wrap with
+                | None -> FlexWrap.NoWrap
+                | Some v -> v
+
+            CollectionBuilder<'msg, IFlexLayout, IView>(
+                FlexLayout.WidgetKey,
+                LayoutOfView.Children,
+                FlexLayout.Wrap.WithValue(wrap)
```

Comment:
It might be better not to add an attribute  if there is no wrap argument was specified